app/views/bracks/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="map-box">
+  <div class="search">
+    <%= form_tag :bracks,  method: :get do %>
+        <%= text_field_tag :q, "Find nearest bike rack...", class: 'form-control' %>
+    <% end %>
+  </div>
+  <hr>
+  <div id="map">
+
+  </div>
+</div>
+
+
+<script>
```

Comment:
Ugh, so this has been annoying and difficult, I'm not sure I fully understand the things that google needs me to do so I'll try again soon, but agree it shouldn't be sitting in the script tags.